docs/commands/bootstrap/README.md

```diff
@@ -30,6 +32,12 @@ $ kam bootstrap
   [--commit-status-tracker]
 ```
 
+## Interactive mode
+
+Running the bootstrap command without flags will trigger an interactive prompt. Each question in the prompt is accompanied by details, thereby providing a brief explanation for that question. 
```

Comment:
`s/details/help/` maybe? and we could drop the _thereby_ ?

Oh, and `s/that question/the question/`

Reply:
I've updated the sentence